File: 프로그래머스/lv0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
import java.util.*;

class Solution {
    public String solution(String s) {
        char[] ch = s.toCharArray();
        Arrays.sort(ch);
        String answer = "";
        for (int i = 0; i < ch.length; i++) {
            if (i != ch.length - 1 && ch[i] == ch[i + 1]) {
                while (i != ch.length - 1 && ch[i] == ch[i + 1]) {
                    i += 1;
                }
            }
            else {
                answer += Character.toString(ch[i]);
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    if len(s) == 1: return s
    answer = ""
    s = sorted(list(s))
    i = 0
    while i < len(s):
        if i != len(s) - 1 and s[i] == s[i + 1]:
            while i != len(s) - 1 and s[i] == s[i + 1]:
                i += 1
        else:
            answer += s[i]
        i += 1
    return answer